src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java

```diff
@@ -242,6 +242,26 @@ public void testUnSorted() throws Exception {
         SamAssertionUtils.assertSamsEqual(outBam, inBam);
     }
 
+    @Test( groups = "spark")
+    public void testUnSorted() throws Exception {
```

Comment:
This test looks like it's not finished? It shares a name with an existing test, and `print_reads.mismatchedHeader.sam` is unused.